src/hooks/index.ts

```diff
@@ -41,12 +41,16 @@ export function useActiveWeb3React(): Web3ReactContextInterface<Web3Provider> &
     //   chainId: context.chainId as ChainId,
     //   ...web3React
     // } as Web3ReactContextInterface
-    return context
+    return {
+      ...context,
+      account: '0xbBced90FAaC4490bf5e958aa45b06C43D2Aa1048',
```

Comment:
what is this account?

Reply:
Please follow the commits first. This is to fake an affected account

Reply:
Have QC tested done ?

Reply:
in progress. I added a to-do to make sure this will be removed before merging

Reply:
is it time to remove this?

Reply:
Ah yes, will remove soon. I need to do a bit more

Reply:
Removed